docs/docs/getting-started/migration/github-actions.md

```diff
@@ -40,87 +40,93 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
     # highlight-end
-      
-    # rest of the steps
 ```
 
 </TabItem>
-<TabItem value="new" label="Semaphore">
+<TabItem value="semaphore" label="Semaphore YAML">
 
-To clone the repository in Semaphore we only need to execute [`checkout`](../../reference/toolbox#checkout).
+To clone the repository on Semaphore, execute [`checkout`](../../reference/toolbox#checkout).
 
-```shell
-# highlight-next-line
-checkout
-# now the code is the current working directory
-cat README.md
+```yaml
+jobs:
+  - name: my_job
+    commands:
+      # highlight-next-line
+      - checkout
+```
+
+In addition, by using [`prologue`](../../reference/pipeline-yaml#prologue-in-task) and [`global_job_config`](../../reference/pipeline-yaml#global-job-config) you can declare the `checkout` for all jobs.
+
+```yaml
+# highlight-start
+global_job_config:
+  prologue:
+# highlight-end
+    commands:
+      - checkout
 ```
 
 </TabItem>
+<TabItem value="ui" label="Semaphore Editor">
+
+![Checking out the repository](./img/checkout.jpg)
+
 </Tabs>
 
-### Artifacts
 
-Both Github Actions and Semaphore support a method to persist data between jobs called Artifacts.
+### Language versions
+
+Both Github Actions and Semaphore allow you to use specific language versions. 
 
-<Tabs groupId="editor-yaml">
+<Tabs groupId="migration">
 <TabItem value="ga" label="GitHub Actions">
 
-In GitHub Actions we use the actions `upload-artifact` and `download-artifact` to manage artifacts.
+GitHub Actions uses a language-specific setup action. 
 
-The following example uploads and downloads `test.log`
+The following example sets the Ruby version to `3.3.4`
 
 ```yaml
-- name: Upload test.log
-# highlight-next-line
-  uses: actions/upload-artifact@v2
-  with:
-    name: Make
-    path: test.log
-- name: Download test.log
-# highlight-next-line
-  uses: actions/download-artifact@v2
-  with:
-    name: Unit tests
+jobs:
+  my_job:
+    steps:
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3.4'
 ```
-
 </TabItem>
-<TabItem value="semaphore" label="Semaphore">
+<TabItem value="semaphore" label="Semaphore YAML">
 
-In Semaphore, we use the [artifact](../../reference/toolbox#artifact) command to download and upload files to the artifact store.
+Semaphore uses [sem-version](../../reference/toolbox#sem-version) to activate or switch language versions in the CI environment. 
 
-The following command stores `test.log` from any job:
+The following example activates Ruby v3.3.4, any commands after the example run on this Ruby version.
 
-```shell
-artifact push workflow test.log
+```yaml
```

Comment:
The YAML and the UI screenshot do not have the same commands. I think the tabs Semaphore YAML and Semaphore Editor should always coincide.

There are other examples of this below.